rag_api/main.py

```diff
@@ -128,11 +152,25 @@ async def load_model(doc_path: str, doc_filter: str):
     return {"status": "success", "doc_path": doc_path, "doc_count": doc_count}
 
 
+@chain
+def retriever_with_scores(query: str) -> List[Document]:
+    """Retrieve documents from vectorstore with similarity score."""
+    # https://python.langchain.com/docs/how_to/add_scores_retriever/
+    docs, scores = zip(*vector_store.similarity_search_with_score(query))
```

Comment:
Shall we increase the num of search result (k) from 4 (default) to like 10?

```suggestion
    docs, scores = zip(*vector_store.similarity_search_with_score(query, k=10))
```

Reply:
Suggestion implemented via settings.CONTEXT_DOCUMENTS_RETRIEVED